financialaid/views_test.py

```diff
@@ -498,6 +472,20 @@ def test_get_learner_price_for_course_allowed(self):
         expected_response = {
             "has_financial_aid_request": False,
             "course_price": self.course_price.price,
-            "financial_aid_adjustment": False
+            "financial_aid_adjustment": False,
+            "financial_aid_availability": True
+        }
+        self.assertDictEqual(resp.data, expected_response)
+        # Enrolled but program has no financial aid availability
+        self.program.financial_aid_availability = False
+        self.program.save()
+        resp = self.assert_http_status(self.client.get, self.course_price_url, status.HTTP_200_OK)
+        expected_response = {
+            "has_financial_aid_request": False,
+            "course_price": self.course_price.price,
+            "financial_aid_adjustment": False,
+            "financial_aid_availability": False
         }
         self.assertDictEqual(resp.data, expected_response)
+        self.program.financial_aid_availability = True
```

Comment:
It will run once per class but it runs within a transaction which is rolled back after the test ends. You only need to run `self.program.refresh_from_db()` in `setUp()` if you are making use of the cached value of `self.program`